kfs-core/src/main/java/edu/arizona/kfs/tax/businessobject/IncomeType.java

```diff
@@ -0,0 +1,66 @@
+package edu.arizona.kfs.tax.businessobject;
+
+import java.util.LinkedHashMap;
+import org.kuali.rice.core.api.mo.common.active.MutableInactivatable;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class IncomeType extends PersistableBusinessObjectBase implements MutableInactivatable {
+    private String incomeTypeCode;
+    private String objectId;
```

Comment:
Property objectId is not required as it is inherited from base class. PersistableBusinessObjectBase.